src/webgl/texture.js

```diff
@@ -112,6 +112,13 @@ export const TEXTURE_FORMATS = {
   // [GL.COMPRESSED_RGBA_ATC_INTERPOLATED_ALPHA_WEBGL]: {compressed: true, gl1: ETC1}
 };
 
+const V3_TO_V4_OPTIONS_MAP = {
```

Comment:
@ibgreen , in v3 branch, setParameters() has following arguments:
```
setParameters({
    magFilter,
    minFilter,
    wrapS,
    wrapT,
    // WEBGL2
    wrapR,
    baseLevel,
    maxLevel,
    minLOD,
    maxLOD,
    compareFunc,
    compareMode
  }) {
```
Assuming I should add all of them, can you please confirm?